

#Camera parameters of depth camera
#All values are of type double
focal_x: 614.6654
focal_y: 614.5995
c_x: 326.94885
c_y: 232.761398

#Thresh hold values for pass-through filter which is used to remove points close and far away
#double
z_min_val: 0.3
z_max_val: 1.4

#Plane segmentation
# accepted distance of 3d points to the plane model to be considered inliers, in [m]
# float
floor_plane_inlier_distance: 0.02
floor_search_iterations: 3

# minimum number of points that are necessary to find the floor plane
# int
min_plane_points: 100


#Euclidean Clustering
#double
cluster_tolerance: 0.03
#int
min_cluster_size: 10

#Cylinder parameters
#double
cylinder_distace_threshold: 0.4

#double
cylinder_radius: 0.25

#Using PCL function to remove outliers
#boolean
use_remove_outliers: true

#number of grasping point
num_grasping_points: 8
# The ground truth height of cylinder object whichs is used to filter out incorrect detection (in meters)
#double
ground_truth_cylinder_height: 0.315 #0.41
# Use cylinder height or not
#boolean
check_ground_truth_cylinder_height: true
# Use direction of cylinder as direction of plane
use_direction_plane: false





#debug display switches - enable or disable the display of several kinds of images, point clouds
# all flags are of type bool


# publishes the detected floor plane
publish_floor_plane: true

# publishes the cylinder object
publish_cylinder_object: true

# shows original color image and the 3D bounding box of the cylinder object
show_cylinder_image: true #false

#show cylinder marker in RVIZ
show_cylinder_marker: true #false




